AngularLanguageService.Shared/LanguageServer/LanguageClient.cs

```diff
@@ -93,6 +93,7 @@ async Task<Connection> ILanguageClient.ActivateAsync(CancellationToken token)
                 RedirectStandardInput = true,
                 RedirectStandardOutput = true,
                 Arguments =
+                    //"--inspect-brk=9222 " +
```

Comment:
I'm guessing this comment is not intended to be here.

Reply:
Yeah but I'll leave it since it's a good reminder on how to debug the server.